extractor/src/main/java/org/schabi/newpipe/extractor/utils/Utils.java

```diff
@@ -52,15 +66,16 @@ public static String removeNonDigitCharacters(final String toRemove) {
      * @param numberWord string to be converted to a long
      * @return a long
      */
-    public static long mixedNumberWordToLong(final String numberWord) throws NumberFormatException,
-            ParsingException {
+    public static long mixedNumberWordToLong(final String numberWord)
+            throws NumberFormatException, ParsingException {
         String multiplier = "";
         try {
             multiplier = Parser.matchGroup("[\\d]+([\\.,][\\d]+)?([KMBkmb])+", numberWord, 2);
         } catch (final ParsingException ignored) {
         }
-        final double count = Double.parseDouble(
-                Parser.matchGroup1("([\\d]+([\\.,][\\d]+)?)", numberWord).replace(",", "."));
+
+        final double count = Double.parseDouble(Parser.matchGroup1("([\\d]+([\\.,][\\d]+)?)",
+                numberWord).replace(",", "."));
```

Comment:
I'm pretty sure that your formatter is not the default Android studio/IntelliJ formatter...

Could you please not format such stuff and use the defaults?

Reply:
> your formatter is not the default Android studio/IntelliJ formatter

My formatter is myself.